walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +313,176 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void optionalAttendeeAlwaysBlocked() {
```

Comment:
(nit) The test name should ideally show both condition (optionalAttendeeAlwaysBlocked) and what is expected as a result (only time slots with mandatory attendees are found?).